### PR TITLE
Week1

### PR DESCRIPTION
**Pull request by MysticRiver**
[Week 1 assignment. WIP as of Monday 11/4]

### CODE REVIEW

**Comment by MysticRiver** [comment]
Pull request for assignment

File: 01-node-tutorial/answers/03-modules.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+const names = require("./04-names.js");
+const sayHi = require('./05-utils')
+const data = require('./06-alternative-flavor')
```
Comment (jonathan-david-johnson) [inline 01-node-tutorial/answers/03-modules.js:3]: On this import i'm getting the following error
```
/workspaces/node-express-course/01-node-tutorial/answers/06-alternative-flavor.js:5
  console.log(upperCase("hello"));
          ^

ReferenceError: upperCase is not defined
    at Object.<anonymous> (/workspaces/node-express-course/01-node-tutorial/answers/06-alternative-flavor.js:5:11)
    at Module._compile (node:internal/modules/cjs/loader:1469:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1548:10)
    at Module.load (node:internal/modules/cjs/loader:1288:32)
    at Module._load (node:internal/modules/cjs/loader:1104:12)
    at Module.require (node:internal/modules/cjs/loader:1311:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (/workspaces/node-express-course/01-node-tutorial/answers/03-modules.js:3:14)
    at Module._compile (node:internal/modules/cjs/loader:1469:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1548:10)

Node.js v20.17.0
```

File: 01-node-tutorial/answers/07-mind-grenade.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const num1 = 10
+const num2 = 5
+
+function DivideNumbers() {
+  console.log(`10 divided by 5 :  {num1 / num2}  `)
```
Comment (jonathan-david-johnson) [inline 01-node-tutorial/answers/07-mind-grenade.js]: Running this file prints the following
```
10 divided by 5 :  {num1 / num2} 
```

**Review by jonathan-david-johnson — COMMENTED** [review]
Mostly good. 

I found a couple errors that shouldn't be too hard to address.

File: 01-node-tutorial/answers/03-modules.js (hunk 1 above)
Comment (MysticRiver) [reply 01-node-tutorial/answers/03-modules.js:3]: I updated the code. Let me know if it works now.

File: 01-node-tutorial/answers/07-mind-grenade.js (hunk 2 above)
Comment (MysticRiver) [reply 01-node-tutorial/answers/07-mind-grenade.js]: I updated the code. Let me know if it works now.

**Review by jonathan-david-johnson — APPROVED** [review]
Great work!

**Comment by MysticRiver** [comment]
Resolved issues raised by mentor. Completed all modules in assignment.

**Review by jonathan-david-johnson — APPROVED** [review]
Looks good.

**Pull request merged**